packages/react/src/components/Flex/FlexGap.tsx

```diff
@@ -0,0 +1,6 @@
+import * as React from 'react'
+
+const FlexGap: React.FC<any> = ({ className }) => <div className={className} />
+FlexGap.displayName = 'FlexGap'
```

Comment:
Or maybe we can define the className for the gap here?

Reply:
this would require to make it `UIComponent` (as it contains logic to handle class names), which it shouldn't be